src/Beberlei/Metrics/Collector/Librato.php

```diff
@@ -77,9 +77,11 @@ public function flush()
             return;
         }
 
+        $authToken = base64_encode($this->username.':'.$this->password);
+
         try {
             $this->browser->post('https://metrics-api.librato.com/v1/metrics', array(
-                'Authorization: Basic '.base64_encode($this->username.":".$this->password),
+                'Authorization: Basic '.$authToken,
```

Comment:
![as you wish](http://www.pop-cultured.net/wp-content/uploads/2015/05/as_you_wish_cover.jpg)